Inheritance wallet - Part 1

rabbitholiness opened the pull request:
This multi-page reference design looks at a multi-key wallet for family savings. It includes a recovery path as well as a timelocked inheritance key set. Apart from wallet creation, the reference design also addresses topics like wallet backups, key rotation and recovery.

This PR addresses the first part of the series, which will cover the landing page, the use case description and the wallet creation flow. The outline of the entire reference design is described on the landing page and marked as "coming soon". The remaining pages will be added step by step. 

👉 Preview it [here](https://deploy-preview-1081--bitcoin-design-site.netlify.app/guide/inheritance-wallet/) 👈

mouxdesign commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,207 @@
+   link-url = "/guide/inheritance-wallet/introduction/"
+%}
+
+Before we dive into the details, we will introduce you to our main protagonists, the Jones family. We are also going to look at how they plan set up their family savings in general. 
```
```suggestion
Before we dive into the details, we will introduce you to the Jones family. We are also going to look at how they plan set up their family savings in general. 
```

mouxdesign commented on guide/inheritance-wallet/introduction.md:
```diff
@@ -0,0 +1,123 @@
+   height = 800
+%}
+
+Note that this configuration could be expanded almost at will by the users. But with each additional layer of logic comes additional complexity in terms of backup and maintainability. While the decision will ultimately be up to users of your application. But you will need to make sure that your application guides and educates them towards choosing and maintaining a setup that is secure enough and ...
```
```suggestion
While this configuration can be expanded by users, every expansion would require an additional layer of complexity. The complexity would enter in terms of backup and maintenance.  Ensuring that the application guides and educates them towards a secure and sustainable setup is essential. 
```

rabbitholiness replied:
Much better than my initial text. Thanks!

mouxdesign reviewed (commented):
Looking good, added in some minor copy suggestions.

rabbitholiness commented:
I'm taking the PR out of draft mode to gather initial feedback. I'll be adding at least one clickable prototype for the wallet creation page by Thursday.

yashrajd commented on assets/images/guide/inheritance-wallet/wallet-creation/creation-inheritance-empty.png:
Nit: is the "Add inheritance key set" card with the '+' icon also a button? It looks like one to me but it might be just me. Perhaps the Continue button can be renamed?

yashrajd commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,290 @@
+
+## Adding the signing keys
+
+The wallet is now fully configured. What is left to do is to add the necessary amount signing keys.
```
Typo here.

```suggestion
The wallet is now fully configured. What is left to do is to add the necessary amount of signing keys.
```

yashrajd commented on assets/images/guide/inheritance-wallet/wallet-creation/creation-request-key-share.png:
Question: how does Alice/Christina scanning this QR allow Bob to request the key?

Fascinating if so, would love to understand...

rabbitholiness replied:
The idea is that "requesting" a key is really just sharing the QR-encoded wallet details. Each request contains a unique identifier that prevents the same invite to be used twice. It also defines the key set to which the XPUB will be imported (primary or inheritance) and potentially the slot, where it should go.

The idea was that the app would keep track of this locally on the wallet creator's app and keep adding the XPUBs to the assigned slots as they come in. Not sure if that is technically feasible, or if there would be another way to do it, but it sounds plausible to me 😂

yashrajd commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,290 @@
+      alt:
+      caption:
+images_add-inheritance-keys: 
+    - file: wallet-creation/creation-request-key-share
```
Think you mean to use Christina's key request QR image here.

```suggestion
    - file: wallet-creation/creation-request-key-share-1
```

yashrajd replied:
I meant this change as a fix to the mistake in the image below (if it is even a mistake). Do confirm this fixes it on your side, I did not test this on local.

<img width="963" alt="dscdfg" src="https://github.com/BitcoinDesign/Guide/assets/114083781/b3780fb6-7702-4900-9ed6-2338ec5dee7f">

yashrajd commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,290 @@
+
+{% include image-gallery.html pages = page.images_add-inheritance-keys %}
+
+An added benefit of everybody using our application from the beginning is that it allows the family to conveniently perform key checks remotely and in regular intervals. 
```
Nit for grammar.

```suggestion
An added benefit of everybody using our application from the beginning is that it allows the family to conveniently perform key checks remotely and at regular intervals. 
```

yashrajd commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,290 @@
+
+## Finalizing the wallet
+
+After all of the keys have been added, Bob and Alice can proceed to create the wallet. They review it the details and hit "create wallet". 
```
Nit: extra word.

```suggestion
After all of the keys have been added, Bob and Alice can proceed to create the wallet. They review the details and hit "create wallet". 
```

yashrajd commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,290 @@
+
+{% include image-gallery.html pages = page.images_homescreen %}
+
+Christina, David and Edward only see that they hold one of the inheritance keys for the family savings wallet. But they cannot see the wallet balance or their parents public keys. If they did, they could start monitoring transactions.
```
Nit: grammar
```suggestion
Christina, David and Edward only see that they hold one of the inheritance keys for the family savings wallet. But they cannot see the wallet balance or their parents' public keys. If they did, they could start monitoring transactions.
```

yashrajd reviewed (changes requested):
Overall, this is fantastic work with amazing structure and eye for detail.

A couple of suggestions, corrections and a bunch of nits. Hope it's useful.

GBKS reviewed (changes requested):
Really nice to see this come together. Lots of little things all around still to tweak.

My main feedback would be around clarifying what role the wallet provider plays, which really affects the user flows. If they handle communication, things are smoother for users. But it also introduces privacy considerations, possibly puts a regulatory burden onto the provider, and can lead to situations where users don't properly take care of things because they rely on the providers free service.

If this is about generational security, then it might be worth minimizing reliance on the provider as a communication middleman.

rabbitholiness commented on guide/inheritance-wallet/introduction.md:
```diff
@@ -0,0 +1,132 @@
+
+Below is an overview of what the configuration of the family's savings wallet will look like:
+
+- **A 2-of-3 primary key set for normal spending**. Since it is a savings wallet, Alice and Bob will spend only very infrequently.
```
Good question. My plan was to not touch the topic in too much detail. It's already a pretty long page. 

That being said. My idea was that each of them store their own key independently. The third key is stored in a shared bank deposit box.

rabbitholiness replied:
Agree. But this is a decision that every family will have to take on their own, based on their specific situation. Plus, the recovery path even lowers the quorum after 6 months. 

Of course, if the relationship between the parents is not a very good one, they might go a different route. They probably would not have a shared savings wallet in the first place, but keep their savings separate, just like before. They also would not have a recovery path set up.

It all comes down to personal choice and what the perceived risk factors are for the users. In this case, the main risk is perceived as self-inflicted loss. The goal of the reference design is to describe the UX.

That being said, it could be a good idea to create a child page under the [Bitcoin backups](https://bitcoin.design/guide/how-it-works/backups/) page, where we could describe a couple of scenarios in more detail 🤔.

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,289 @@
+
+The inheritance keys are added in the same way. Christina, David and Edward also use their own phones to add their signing devices to the wallet. Alternatively, they could have transmitted their public keys to Alice and Bob over a secure channel.
+
+In the background, our application marks their keys as inheritance keys. The app then keeps track of whether the inheritance key set is unlocked or not and manages wallet access accordingly, on the application layer. This allows David, Christina and Edward to participate in the setup from the beginning without compromising the financial privacy of the parents. 
```
The idea is that as long as the inheritance key set is locked, the heirs only see in the app that they hold one of the inheritance keys. As soon the inheritance key set is unlocked, they have the same view as the wallet "owners" and can spend funds.

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,289 @@
+
+An added benefit of everybody using our application from the beginning is that it allows the family to conveniently perform key checks remotely and at regular intervals. 
+
+On top of that, the children don't have to set up anything more or run through a dedicated recovery process when the time comes. The app just unlocks the wallet so that they can move the funds to another wallet. 
```
As soon the inheritance key set is unlocked, they heirs have the same view as the wallet "owners" and can spend funds.

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,289 @@
+
+{% include image-gallery.html pages = page.images_primary-keys-bob %}
+
+Our application allows the users to create the wallet collaboratively. This means that each key holder can use their own phone and signing device to add their key. So Bob creates a key request that Alice can use.
```
That's a very good point. Thanks for bringing it up!

My original idea was that the app is optimized for collaborative wallet creation. The idea is that "requesting" a key is really just sharing the QR-encoded wallet details. Each request contains a unique identifier that prevents the same invite from being used twice and also defines the key set to which the XPUB will belong (primary or inheritance). 

During that process, the app (wallet provider) would keep track of this and keep adding the XPUBs to the assigned key set as they come in. You're right that this implies the wallet provider of storing the configuration somewhere, until wallet creation is finished. Since there is no signup required, the wallet provider would only see the configuration of the wallet with no additional data. But, as you point out, it introduces risk for the users, as the provider can tamper with the wallet config. 

Might well be that I'm over-emphasizing the asynchronous and remote use case. After all, meeting physically (or remotely over video call) is probably a reasonable thing to do in a family.

rabbitholiness commented on guide/inheritance-wallet/introduction.md:
```diff
@@ -0,0 +1,123 @@
+# Use case & scope 
+{: .no_toc }
+
+Our target audience are families that are familiar with bitcoin and hold the majority of their net worth in bitcoin. They want to take advantage of the security assurances of multi-key wallets and are comfortable with using more advanced technical features and human processes for inheritance planning.  
```
Thanks for the suggestion. I formulated it slightly differently.

mouxdesign commented:
Just reviewed the wallet creation page, looks great, really like how vibrant and friendly the designs look.

Some ideas:
The user flow itself is split into 3 distinct sections, each of those having subsections under each. As a builder landing on this page the flow is pretty detailed. To manage the expectations of the builder, if they commit to building this out in their wallet then how much of complexity would it require. See that there is a high level menu at the top which allows the reader to navigate however that would be more to navigate the content. A higher level overview could help with this in the beginning, this can be an image or text setting expectations:

**Step1: First configure wallet**
1. Primary keyset (add keys)
2. Create recovery path
3. Create dedicated inheritance keyset
Are any of these steps optional? 

**Step 2: Add signing keys**
1. Primary
2. Inheritance keys

**Step 3: Finalize the wallet**

Spelling:
there was one small typo on one of the screen designs:
![Screenshot 2024-04-22 135736](https://github.com/BitcoinDesign/Guide/assets/90313394/83c67a02-fefd-4c39-aca4-c4797d82f8ec)
Custom "answer a couple of questions"

Other than that the language on the page is nice and easy to understand, you did a really great job of that considering how complex this topic is.

rabbitholiness commented:
Just pushed a quite extensive update. Here's the TL;DR:

**Use case & scope page:**
- Clarified and elaborated on the role that the wallet provider plays as well as the approach taken in this reference design, based on @GBKS feedback.
- Structured the second half of the page into a "Design considerations" section.

**Wallet creation page:**
- Added more structure to the page content, based on @mouxdesign feedback.
- Updated designs and image galleries for adding signing keys. They now reflect the non-custodial approach of the application. The original designs are still available in Figma. 
- Added captions and missing alt texts.

danielnordh reviewed (commented):
Apologies for late review.
Mostly minor comments but some broken links.

Introductory page

- Clicking on "Use case & scope" on main page goes to 404, also when when clicking the next button at bottom of page, sidebar works as expected
- Consider renaming "Use case & scope" to "Family use case" or "Overview"?

Use case & scope page

- "A timelocked recovery path in case they lose two keys and need to recover their funds. The recovery path uses the same keys as the primary key set, but requires only one signature to spend." could be less ambiguous: "A timelocked recovery path in case they lose two of the three keys. The timelock enables moving the funds with just one of the keys if no signing has occurred within a set time, in this case we'll set it to 6 months." 
- "The image below illustrates how the wallet will work under the hood." You could use the image caption feature instead: caption = "Text goes here."
- Use "self-custodial" instead of "non-custodial"
- Should miniscript be capitalised? Could also have a link to relevant page.

Wallet creation page
- Very minor: Image captions under some of the onboarding screens gets long, stick to label/description and not 'explanation' (as screenshot does a good job)
- Very minor: Emojis steal attention from the header on each onboarding screen
- Minor: Recovery path screenshots, I would show the time immediately to make it clear that the recovery is time dependent, as it is not mentioned in the text
- Minor: instead of "Recovery path" I wonder if there is something more user friendly; Recovery option, Single key recovery, Time based recovery... 
- Maybe 'amount' is not meant to be in this sentence?: "What is left to do is to add the necessary amount signing keys."
- The 'next page' button at the bottom links to Upgradeable wallet

rabbitholiness commented on guide/inheritance-wallet/introduction.md:
```diff
@@ -0,0 +1,132 @@
+- Make changes to the setup over time AND make sure that its documentation is updated and distributed accordingly.
+- Provide a way to migrate to a different wallet application, in case your project is abandoned or shut down.
+
+Such supporting features can be implemented on the application layer and don't necessarily need to touch the bitcoin protocol.
```
Removed the second part of the sentence, as it was superfluous.

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,289 @@
+
+#### Recovery path
+
+After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. We have covered this in the time-based reference design as well. Here we show an alternative design option for this same feature.
```
Removed the sentence, as it as not relevant

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,289 @@
+
+#### Inheritance key set
+
+After enabling the recovey path, the app offers users to create a dedicated inheritance key set. This works in more or less the same way as for the primary key set.
```
Tweaked the text for more clarity.

rabbitholiness commented on guide/inheritance-wallet/introduction.md:
```diff
@@ -0,0 +1,132 @@
+   height = 800
+%}
+
+Note that this configuration could be expanded almost at will by the users. But with each additional layer of logic comes additional complexity in terms of backup and maintainability. The decision will ultimately be up to users of your application. But you will need to make sure that your application guides and educates them towards choosing and maintaining a setup that is both, secure enough and ...
```
I changed the the text and moved it under a new "Design considerations" section, to make it more distinct.

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,290 @@
+
+We could also call the recovery path a recovery key set, since it technically is just that. However, the point of the recovery path is that we want to use the same keys that are used in the primary key set. To make this distinction more clear, we treat recovery paths as a different feature from alternative key sets in our application. You might want to take a different approach in your own applic ...
+
+#### Inheritance key set
```
Rephrased this part. I hope it's clearer now. I feel like the distinction between primary key set and inheritance key set is not a bad one. It makes it clear that they are different keys. The recovery path is a way to recover funds using the primary key set.

rabbitholiness commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,289 @@
+
+Adding a key technically means that users have to import the extended public key (XPUB) from each signing device that should be used to sign transactions. 
+
+This sounds more complicated than it actually is. Depending on the specific signing devices that are being used, it can be as simple as scanning a QR code, connecting it via NFC or Bluetooth, or by using a USB cable. Bob does just that and imports the first two keys by himself.
```
No, the idea is that all keys are external keys:
- Bob's Trezor
- Alice's Bitbox
- A ColdCard stored in a safety deposit box to which both, Alice and Bob, have access.

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/header.png
+image_base: /assets/images/guide/inheritance-wallet/
```
Can delete the image_base line if you don't use the `_image-gallery` partial anywhere. You can also remove the `images_creation-wallet-basics:` object right underneath if you don't use us.

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+
+Editor's notes
+
+This page covers ....  
```
What does it cover? Editor's notes are for people who might edit this file later to better understand editorial decisions made during writing, that might not be obvious.

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+   image = "/assets/images/guide/inheritance-wallet/header.jpg"
+   retina = "assets/images/guide/inheritance-wallet/header@2x.jpg"
+   alt-text = "Two parents handing bitcoin to their two children."
+   width = 1600
```
Width and height do not match the `header.jpg` file size. Either adjust the width and height here, or the image size. I'd suggest what I mentioned in a comment above, to have a separate preview and header file.

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+# Family inheritance wallet 
+{: .no_toc }
+
+The family inheritance wallet is an app designed to create flexible wallets for family savings. It offers features designed to help with sustainable long-term storage and inheritance planning.
```
Considering [the situation](https://stacker.news/items/571729) we had recently where someone posted the daily spending wallet to Stacker News and people thought it was as real wallet, do we need to include the term `reference design` somewhere in this intro sentence? Not sure if it would make a difference...

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+
+On top of that, the app allows you to add your heirs' hardware wallets and use them to create a dedicated inheritance key set. Since it is designed for long-term storage, it also includes regular key checks and other supporting features. 
+
+All of this is done without compromising your financial privacy. You don't want anyone to know how much bitcoin you have or monitor your transactions.
```
```suggestion
All of this is done without compromising your financial privacy. You don't want anyone to know how much bitcoin you have or to monitor your transactions.
```

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+   link-url = "/guide/inheritance-wallet/introduction/"
+%}
+
+Before we dive into the details, we will introduce you to the Jones family. We are also going to look at how they plan set up their family savings in general. 
```
```suggestion
Before we dive into the details, we will introduce you to the Jones family. We are also going to look at how they plan to set up their family savings in general. 
```

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+   link-url = "/guide/inheritance-wallet/backup/"
+%}
+
+When backing up multi-key wallets there are more moving parts to consider than with single-key wallets. Our application helps users with that. 
```
```suggestion
When backing up multi-key wallets, there are more moving parts to consider than with single-key wallets. Our application helps users with that. 
```

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+   link-url = "/guide/inheritance-wallet/regular-use/"
+%}
+
+Involving the potential heirs early on is great, but it also has potential implications when it comes to privacy and security. Our application combines built-in features with social processes to mitigate this risk. It also helps to prevent premature activation of the inheritance keys. 
```
```suggestion
Involving the potential heirs early on is great, but it also has implications for privacy and security. Our application combines built-in features with social processes to mitigate this risk. It also helps to prevent premature activation of the inheritance keys. 
```
Tweak for simplicity.

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+   link-url = "/guide/inheritance-wallet/managing-change/"
+%}
+
+Over time it might become necessary for users to make changes to their savings wallet because someone lost a key or they want to make changes to the overall wallet configuration. Our app helps users to make such changes and update the backup and inheritance documentation.
```
```suggestion
Over time, it might become necessary for users to make changes to their savings wallet because someone lost a key or they want to make changes to the overall wallet configuration. Our app helps users make such changes and update the backup and inheritance documentation.
```

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,208 @@
+--- 
+
+**Resources**
+- All illustrations, mockups and interactive prototypes are available in this [Figma design file](https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1)
```
Consistency: Add period to end of both bullets or none.

GBKS replied:
```suggestion
- All illustrations, mockups and interactive prototypes are available in this [Figma design file](https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1).
```

GBKS commented on guide/inheritance-wallet/overview.md:
```diff
@@ -0,0 +1,133 @@
+
+The target audience for the inheritance wallet are families who are familiar with bitcoin and hold the majority of their net worth in bitcoin. They want to take advantage of the security assurances of multi-key wallets and are comfortable with using more advanced technical features and human processes for inheritance planning.  
+
+For other users, such a setup might be too advanced. When choosing a specific setup, the decision should be based on factors such as the amount of bitcoin they hold, what their family situation looks like, where they live or how knowledgeable and comfortable they are with bitcoin in general. This line of thinking applies not only to the initial bitcoin holders themselves, but also to their heirs, ...
```
We could link to the [personal finance page](https://bitcoin.design/guide/designing-products/personal-finance/), which provides a framework for choosing the right security setup for your use case.

Reminder to self to update the personal finance page at some point with a link to this inheritance wallet reference design.

GBKS replied:
Also, the first sentence is very long. Considering our international audience with many for whom English is the second language, I think it's good to keep text short and direct. Up to you whether you want to adjust it or not.

GBKS commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,338 @@
+
+Now that we have a [good understanding]({{ '/guide/inheritance-wallet/overview/' | relative_url }}) of how Bob and Alice want to set up their inheritance wallet, let's take a close look at the first use experience and wallet setup.
+
+After downloading the app, the app walks them through the purpose of the product and highlights some of the most important features. 
```
```suggestion
After downloading the app, it walks them through the purpose of the product and highlights some of the most important features. 
```

GBKS commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,338 @@
+
+## Step 1: Wallet configuration
+
+The first is the wallet configuration, where Bob and Alice define the rules for how the wallet should work. In the second part, they add all the signing keys.
```
```suggestion
We start with wallet configuration, where Bob and Alice define the rules for how the wallet should work. In the second part, they add all the signing keys.
```

GBKS commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,338 @@
+
+#### Enabling the recovery path
+
+After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. The recovery path uses the same keys that are part of the primary key set and unlocks after 6 months of inactivity. If it is unlocked, only one signature will be required to spend bitcoin, instead of two. We have covered this in the time-based reference design as well.
```
```suggestion
After configuring the primary key set, the app asks the users whether they would like to enable a recovery path. The recovery path uses the same keys that are part of the primary key set and unlocks after 6 months of inactivity. If it is unlocked, only one signature will be required to spend bitcoin, instead of two. We have covered this in the time-based reference design as well.
```
Turn `time-based reference design` into a link to that page.

GBKS commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,338 @@
+{% include image-gallery.html pages = page.images_primary-keys-bob %}
+
+##### Bob requests the third key from Alice
+Next, it's time to add Alice's key, which will be imported from her Bitbox. Our app makes this process easy, because Bob can just create a key request that Alice can scan with her phone. Alternatively, he can also send it to her via a secure channel.
```
```suggestion
Next, it's time to add Alice's key, which will be imported from her BitBox. Our app makes this process easy, because Bob can just create a key request that Alice can scan with her phone. Alternatively, he can also send it to her via a secure channel.
```

GBKS commented on guide/inheritance-wallet/wallet-creation.md:
```diff
@@ -0,0 +1,338 @@
+During the family reunion, Bob opens our application and resumes the wallet creation process. Just like with Alice's key, he creates a key request for Christina and David.
+
+##### Christina creates a new key in the app
+Christina could follow the same procedure for importing a key from her existing signing device, as described above. However, Christina does not want to use her existing hardware wallet from the family savings. Instead, she wants to create a new key for the family savings wallet, so that she can keep it on a seperate signing device. 
```
```suggestion
Christina could follow the same procedure for importing a key from her existing signing device, as described above. However, Christina does not want to use her existing hardware wallet from the family savings. Instead, she wants to create a new key for the family savings wallet, so that she can keep it on a separate signing device. 
```

GBKS reviewed (changes requested):
Looks really good. Just had a few minor comments.

I also recommend running the images through a compression tool like [ImageOptim](https://imageoptim.com/mac). You can cut the file size by 50-80%.

In the wallet creation page, I think the file `coming-soon-placeholder.png` is not used anymore. Most likely you have more unused images in there. You use 56 visuals in image galleries, which means 112 images. Then there are 4 images for the prototype, for a total of 116. But the wallet creation folder includes 129 images. Maybe another 12 unused ones in there?

rabbitholiness commented:
> Looks really good. Just had a few minor comments.
> 
> I also recommend running the images through a compression tool like [ImageOptim](https://imageoptim.com/mac). You can cut the file size by 50-80%.
> 
> In the wallet creation page, I think the file `coming-soon-placeholder.png` is not used anymore. Most likely you have more unused images in there. You use 56 visuals in image galleries, which means 112 images. Then there are 4 images for the prototype, for a total of 116. But the wallet creation folder includes 129 images. Maybe another 12 unused ones in there?

Should all be addressed now. Took some digging but I found the superfluous images :-)

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,203 @@
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/preview.jpg
+image_base: /assets/images/guide/inheritance-wallet/
```
You can remove the `image_base`. It is only used in galleries to define which folder the gallery images are in. You don't use galleries, so you don't need to set this.

GBKS commented on guide/inheritance-wallet/landing-page.md:
```diff
@@ -0,0 +1,203 @@
+has_children: true
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/preview.jpg
```
You don't need a @2x version of the preview.jpg

GBKS commented on guide/inheritance-wallet/overview.md:
```diff
@@ -0,0 +1,133 @@
+   height = 800
+%}
+
+Alice and Bob have been using a [shared multi-key wallet](https://bitcoin.design/guide/shared-wallet/) as a family checking account for a few years already. This wallet is used for general spending, while each of them manage their income, personal spending and savings using their individual single-key wallets. 
```
```suggestion
Alice and Bob have been using a [shared multi-key wallet]({{ '/guide/shared-wallet/' | relative_url }}) as a family checking account for a few years already. This wallet is used for general spending, while each of them manage their income, personal spending and savings using their individual single-key wallets. 
```

GBKS commented on guide/inheritance-wallet/overview.md:
```diff
@@ -0,0 +1,133 @@
+Below is an overview of what the configuration of the family's savings wallet will look like:
+
+- **A 2-of-3 primary key set for normal spending**. Since it is a savings wallet, Alice and Bob will spend only very infrequently.
+- **A timelocked [recovery path](https://bitcoin.design/guide/savings-wallet/time-based-recovery/)** in case they lose two of the three keys. The timelock enables moving the funds with just one of the keys if no signing has occurred within a set time, in this case we'll set it to 6 months.
```
```suggestion
- **A timelocked [recovery path]({{ 'guide/savings-wallet/time-based-recovery/' | relative_url }})** in case they lose two of the three keys. The timelock enables moving the funds with just one of the keys if no signing has occurred within a set time, in this case we'll set it to 6 months.
```

GBKS commented on guide/inheritance-wallet/overview.md:
```diff
@@ -0,0 +1,133 @@
+
+#### Interoperability
+
+An important aspect to consider is the demise of our application itself. What if, during the course of the years, the developers of our application decide to abandon the project? Or if it gets removed from the app stores? Ensuring a high degree of [interoperability](https://bitcoin.design/guide/getting-started/principles/#interoperability) with protocol standards will allow users to easily move t ...
```
```suggestion
An important aspect to consider is the demise of our application itself. What if, during the course of the years, the developers of our application decide to abandon the project? Or if it gets removed from the app stores? Ensuring a high degree of [interoperability]({{ '/guide/getting-started/principles/#interoperability' | relative_url }}) with protocol standards will allow users to easily move their wallet to a different application.
```

GBKS reviewed (changes requested):
Few more tweaks.

rabbitholiness commented:
@GBKS do you think we can get this one merged?

GBKS reviewed (dismissed):
ACK. Looks great. Thanks for addressing all my nitpicky feedback. Let's get this one out the door.

mouxdesign commented:
Looks good to fly Michael. 

One small comment is that the icon for the wallet backup is not sowing up on my screen. 

![Screenshot 2024-07-01 111709](https://github.com/BitcoinDesign/Guide/assets/90313394/b2062d8c-72e9-4dee-88cf-a52cbcdf2761)

GBKS reviewed (approved):
Re-acking.
